### PR TITLE
Removes some socially charged things from the ion law lists, adds mothmen to the species list

### PR DESCRIPTION
**Pull request by CameronWoof**
## About The Pull Request

Removes "angry black men", "heterosexual", and "homosexual". Adds "mothmen".

## Why It's Good For The Game

Some players downstream mentioned being uncomfortable with the idea of being singled out because of a real-life sexuality or race. I was told to PR it up here first.

## Changelog
:cl:
tweak: Changed around the ion law lists
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
downstreams pushing controversial decisions upstream


https://user-images.githubusercontent.com/2544323/123176651-f34e8a80-d4d7-11eb-8b27-90a04a8df522.mp4

**Comment by Ironnhawk** [comment]
@optimumtact May I suggest another look, I think they worded this poorly, but this is in regards to your rule 11

File: strings/ion_laws.json [hunk 1]
```diff
@@ -837,7 +835,8 @@
         "SLIME PEOPLE",
         "GOLEMS",
         "SHADOW PEOPLE",
-        "CHANGELINGS"
+        "CHANGELINGS",
+        "MOTHMEN"
```
Comment (tralezab) [inline strings/ion_laws.json:839]: erm... you meant to say "MONTHMEN"?

Comment (Ironnhawk) [reply strings/ion_laws.json:839]: No, no no I didnt make this, but the person that did was trying to remove primarily this

Comment (Ironnhawk) [reply strings/ion_laws.json:839]: image

Comment (Ironnhawk) [reply strings/ion_laws.json:839]: image

**Comment by optimumtact** [comment]
@Ironnhawk no you may not

**Pull request closed without merging**